### DISCUSSION TITLE
What are some decor items we should have?

### DISCUSSION THREAD

**Discussion by inactivelaylabmanley** [Boxnet-Media/boxnetjam#35] [Ideas]
After creating #34, I was thinking about additional art assets we'd need for level design.

What kind of objects may we need?

**Comment by MichaPika** [comment]
- tables
- lockers
- boxes
   - crates, storage
- desks
- chairs
- lights
  - lamps, wall lights
- maybe some type of beds or something for different rooms?

Just throwin stuff out haha
